sdk/textanalytics/Azure.AI.TextAnalytics/tests/AnalyzeOperationTests.cs

```diff
@@ -273,25 +288,25 @@ public async Task AnalyzeOperationWithPagination()
 
             // try async
             //There most be 2 pages as service limit is 20 documents per page
-            List<AnalyzeOperationResult> asyncPages = operation.Value.ToEnumerableAsync().Result;
+            List<AnalyzeBatchActionsResult> asyncPages = operation.Value.ToEnumerableAsync().Result;
             Assert.AreEqual(2, asyncPages.Count);
 
             // First page should have 20 results
-            Assert.AreEqual(20, asyncPages[0].Tasks.KeyPhraseExtractionTasks.FirstOrDefault().Results.Count);
+            Assert.AreEqual(20, asyncPages[0].ExtractKeyPhrasesActionsResults.FirstOrDefault().Result.Count);
 
             // Second page should have remaining 3 results
-            Assert.AreEqual(3, asyncPages[1].Tasks.KeyPhraseExtractionTasks.FirstOrDefault().Results.Count);
+            Assert.AreEqual(3, asyncPages[1].ExtractKeyPhrasesActionsResults.FirstOrDefault().Result.Count);
 
             // try sync
             //There most be 2 pages as service limit is 20 documents per page
-            List<AnalyzeOperationResult> pages = operation.GetValues().AsEnumerable().ToList();
+            List<AnalyzeBatchActionsResult> pages = operation.GetValues().AsEnumerable().ToList();
             Assert.AreEqual(2, pages.Count);
 
             // First page should have 20 results
-            Assert.AreEqual(20, pages[0].Tasks.KeyPhraseExtractionTasks.FirstOrDefault().Results.Count);
+            Assert.AreEqual(20, pages[0].ExtractKeyPhrasesActionsResults.FirstOrDefault().Result.Count);
 
             // Second page should have remaining 3 results
-            Assert.AreEqual(3, pages[1].Tasks.KeyPhraseExtractionTasks.FirstOrDefault().Results.Count);
+            Assert.AreEqual(3, pages[1].ExtractKeyPhrasesActionsResults.FirstOrDefault().Result.Count);
         }
 
         [Test]
```

Comment:
this test should be enable now. It is not related to pagination and it is the only one that could check the error logic you implemented.
it will need update too

Reply:
updated